### PR TITLE
fix #282: localize JS strings by using nativeState js-java interface

### LINKED ISSUE #282: Localize JS strings
`Edit` and `Tap to try again` are hardcoded strings in the `ZSSEditor` and cannot be localized. We'll probably need to assign those strings on init, so we can pass a localized String from the native side.

Counterpart issue in the iOS editor repo: https://github.com/wordpress-mobile/WordPress-Editor-iOS/issues/756

### PR DESCRIPTION
**Pull request by maxme**
fix #282: localize JS strings by using nativeState js-java interface

### CODE REVIEW

File: WordPressEditor/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <string name="delete">Delete</string>
     <string name="save">Save</string>
     <string name="discard">Discard</string>
+    <string name="edit">Edit</string>
+    <string name="tap_to_try_again">Tap to try again!</string>
+    <string name="uploading">Uploading…</string>
+    <string name="uploading_gallery_placeholder">[Uploading Gallery]</string>
```
Comment (aforcier) [inline WordPressEditor/src/main/res/values/strings.xml]: Could we keep 'Uploading gallery...' as the message? Also it's probably better to remove the square brackets from the string and add wrap them around the method call in JavaScript instead - that way we can reuse this string in the future if we add visual gallery support and can drop the shortcode in visual mode.

**Comment by aforcier** [comment]
Looks good @maxme, good to :shipit: once that line note is addressed.

**Comment by maxme** [comment]
Fixed in 166cf1e

**Comment by aforcier** [comment]
:shipit:

**Pull request merged**